tests/test_memberships.py

```diff
@@ -48,7 +48,7 @@ class TestMembershipData(unittest.TestCase):
 
     def test_data(self):
         """Test the working group data is clean."""
-        res = Group.parse_obj(yaml.safe_load(OPERATIONS_METADATA_PATH.read_text()))
+        res = Group.model_validate(yaml.safe_load(OPERATIONS_METADATA_PATH.read_text()))
```

Comment:
I cant judge this change :)

Reply:
Its just because `parse_obj` method was deprecated in Pydantic 2.0 and the recommended replacement is `model_validate`.
The functionality still remains the same - the method converts the incoming dictionary into the Group object.

Reply:
Thanks!